src/components/browse-data/trials/TrialTable.tsx

```diff
@@ -347,10 +367,13 @@ export const usePaginatedTrials = (token: string) => {
         setSize(size => size + 1);
     }, [setSize]);
 
-    // Clear all results when filters change
+    // Clear all results when filters change if data has already loaded
+    const hasTrials = !!trials;
     React.useEffect(() => {
-        setSize(1);
-    }, [filters.trial_ids, setSize]);
+        if (hasTrials) {
```

Comment:
Without this check, `usePaginatedTrials` makes a duplicate API request on initial render.